cmd/zt_generic_traverser_test.go

```diff
@@ -198,7 +206,49 @@ func (s *genericTraverserSuite) TestWalkWithSymlinks(c *chk.C) {
 
 	// 3 files live in base, 3 files live in symlink
 	c.Assert(fileCount, chk.Equals, 6)
+	c.Assert(sawLinkTargetDir, chk.Equals, true)
+}
+
+// Next test is temporarily disabled, to avoid changing functionality near 10.4 release date
+/*
```

Comment:
We do need at least _basic_ symlink following for files-- even if it's just that temporary map for handling individual file symlinks.

---
In reply to: [402580645](https://github.com/Azure/azure-storage-azcopy/pull/942#discussion_r402580645) [](ancestors = 402580645)

Reply:
As discussed, it's not a breaking change because following file symlinks was already broken in 10.3.4 ;-)   So let's not complicate things by fixing that now, but let's fix it in, say 10.4.1 or something.